src/pages/managerForm.js

```diff
@@ -1,66 +1,209 @@
-import React from 'react';
-import Link from 'next/link';
-import Image from 'next/image'
-import NavBar from '@/components/Header/NavBar';
-import CustomizableButton from '@/components/common/CustomizableButton';
-import FooterBar from '@/components/common/FooterBar';
-import styles from '../styles/managerForm.module.css';
+import { useState, useRef } from "react";
+import React from "react";
+import { useRouter } from "next/navigation";
+import Link from "next/link";
+import Image from "next/image";
+import NavBar from "@/components/Header/NavBar";
+import CustomizableButton from "@/components/common/CustomizableButton";
+import FooterBar from "@/components/common/FooterBar";
+import { imageUpload, uploadFile, deleteFile, resetInput, goHome } from "@/pages/helperFile";
+import styles from "../styles/managerForm.module.css";
+import { firebaseDb, firebaseStorage } from "@/firebase/config";
+import {collection, addDoc} from "firebase/firestore";
+import { uploadBytes, getDownloadURL, ref } from "firebase/storage";
 
 
-const managerForm = () => {
-    
-    const categoryChoice = () => {
-        console.log("clicked the button")
+function ManagerForm() {
+  const initialFormData = {
```

Comment:
I think this could also be extracted to a ManagerFormHelper file. Not super important but if you are going to create one based on the other comment below then this could go in there too.

Reply:
Due to time constraints, I will defer this until all other issues are done.